sktime/forecasting/compose/_pipeline.py

```diff
@@ -114,6 +76,196 @@ def named_steps(self):
+        return self
+
+
+class ForecastingPipeline(_Pipeline):
+    """
+    Pipeline for forecasting with exogenous data to apply transformers
+    to the exogenous serieses. The forecaster can also be a
+    TransformedTargetForecaster containing transformers to
+    transform y. ForecastingPipeline is only applying the given transformers
+    to X.
+
+    Parameters
+    ----------
+    steps : list
+        List of tuples like ("name", forecaster/transformer)
+
+    Example
+    -------
+    >>> from sktime.datasets import load_longley
+    >>> from sktime.forecasting.naive import NaiveForecaster
+    >>> from sktime.forecasting.compose import ForecastingPipeline
+    >>> from sktime.transformations.series.impute import Imputer
+    >>> from sktime.transformations.series.adapt import TabularToSeriesAdaptor
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> from sktime.forecasting.model_selection import temporal_train_test_split
+    >>> from sklearn.preprocessing import MinMaxScaler
+    >>> y, X = load_longley()
+    >>> y_train, _, X_train, X_test = temporal_train_test_split(y, X)
+    >>> fh = ForecastingHorizon(X_test.index, is_relative=False)
+    >>> pipe = ForecastingPipeline(steps=[
+    ...     ("imputer", Imputer(method="mean")),
+    ...     ("minmaxscaler", TabularToSeriesAdaptor(MinMaxScaler())),
+    ...     ("forecaster", NaiveForecaster(strategy="drift"))])
+    >>> pipe.fit(y_train, X_train)
+    ForecastingPipeline(...)
+    >>> y_pred = pipe.predict(fh=fh, X=X_test)
+    """
+
+    _required_parameters = ["steps"]
+
+    def __init__(self, steps):
+        self.steps = steps
+        self.steps_ = None
+        super(ForecastingPipeline, self).__init__()
+
+    def fit(self, y, X=None, fh=None):
+        """Fit to training data.
+
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        fh : int, list or np.array, optional (default=None)
+            The forecasters horizon with the steps ahead to to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored
+        Returns
+        -------
+        self : returns an instance of self.
+        """
+        self.steps_ = self._check_steps()
+        self._set_y_X(y, X)
+        self._set_fh(fh)
+
+        # transform X
+        y, Xt = check_y_X(y, X)
+        for step_idx, name, transformer in self._iter_transformers():
+            t = clone(transformer)
+            Xt = t.fit_transform(Xt)
+            self.steps_[step_idx] = (name, t)
+
+        # fit forecaster
+        name, forecaster = self.steps[-1]
+        f = clone(forecaster)
+        f.fit(y, Xt, fh)
+        self.steps_[-1] = (name, f)
+
+        self._is_fitted = True
+        return self
+
+    def _predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        Xt = check_X(X)
+        forecaster = self.steps_[-1][1]
```

Comment:
what happens if we only pass transformers without a final forecaster?

Reply:
this is not yet working, I thought I might do it in a separate PR also then for `TransformedTargetForecaster`

Reply:
this has to be done when we accept `isinstance(y, pd.DataFrame)`